File: Clients/BlogBlazor/Shared/Blazor.Contracts/V1/Requests/TagsRequests/UpdateTagRequest.cs
namespace Blazor.Contracts.V1.Requests.TagsRequests;

using Interfaces;

/// <summary>
/// Update tag request.
/// </summary>
public class UpdateTagRequest : IRequest
{
    /// <summary>
    /// Gets or sets the title.
    /// </summary>
    /// <value>
    /// The title.
    /// </value>
    public string Title { get; set; }
}

File: Shared/Blog.FSharp.Contracts/V1/Requests/TagsRequests/UpdateTagRequest.fs
namespace Blog.FSharp.Contracts.V1.Requests.TagsRequests

open Blog.FSharp.Contracts.V1.Requests.Interfaces

/// <summary>
/// Update tag request.
/// </summary>
type UpdateTagRequest() =
    interface IRequest

    /// <summary>
    /// Gets or sets the title.
    /// </summary>
    member val Title : string = null with get, set